Anudle-sy/pest-smart
# Add Mocking Library and Create First Endpoint

Summary:
- added the MSW library (followed example set up [here](https://github.com/vercel/next.js/pull/13731)
- create first endpoint to get all posts (returns fake data - TODO MM still need to set up collections in MongoDB)

Test Plan:
1. Navigate to http://localhost:3000/posts
2. Verify that the test data loads and the page looks like the following:
![image](https://user-images.githubusercontent.com/81973596/116286904-609ea300-a755-11eb-9139-1f88591640ab.png)
